sympy/polys/ring_series.py

```diff
@@ -1898,8 +1898,9 @@ def _rs_series(expr, series_rs, a, prec):
     elif expr.is_Mul:
         n = len(args)
         for arg in args:    # XXX Looks redundant
-            R1, _ = sring(arg, expand=False, series=True)
-            R = R.compose(R1)
+            if not arg.is_Number:
```

Comment:
For the `QQ` ring, the current behaviour of `is_Number` is what we want. But I would like to make the difference more explicit so that if we use a different ring (say `RR`) in future, there is no confusion.

So, I think `is_Rational` is a better choice.
@smichr I presume that `is_rational` includes unevaluated rational and `is_Rational` doesn't.